### PR TITLE
upgrade singleton service to springboot3.1.1

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

**Review by Xiaochao8 — APPROVED** [review]

File: g11n-ws/build.gradle [hunk 1]
```diff
@@ -48,18 +50,17 @@ allprojects{
...
 
-    sourceCompatibility = '1.8'
-     
+
     ext {
    	       
         projectVersion = '0.0.1-SNAPSHOT'
-        springWebVersion='5.3.27'
-        tomcatVersion= '9.0.75'
+        //springWebVersion='6.0.10'
```
[the remark this answers is not in the document]
Comment (tigershi) [reply g11n-ws/build.gradle:57]: done

**Comment by Xiaochao8** [comment]
CI doesn't pass.

**Comment by tigershi** [comment]
it need CI change scripts

**Comment by huihuiw01** [comment]
At the last of the PR, I see several classes in vip-l10n-manager and vip-i18n-lite-manager are deleted, what's the reason?

**Review by Xiaochao8 — DISMISSED** [review]

**Review by Xiaochao8 — APPROVED** [review]

**Comment by tigershi** [comment]
> At the last of the PR, I see several classes in vip-manager-l10n and vip-manager-lite-i18n are deleted, what's the reason?

all delete file is about swagger UI， because of change the springfox to springdoc, we need to change the swagger content

**Review by huihuiw01 — APPROVED** [review]

**Pull request merged**